src/components/Tabs/components/List/tests/List.test.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import {mountWithAppProvider} from 'test-utilities';
+import List from '../List';
+import Item from '../../Item';
+
+describe('<List />', () => {
+  const mockProps = {
+    focusIndex: -1,
+    disclosureTabs: [
+      {
+        id: 'repeat-customers',
+        content: 'Repeat customers',
+      },
+      {
+        id: 'prospects',
+        content: 'Prospects',
+      },
+    ],
+  };
+
+  it('renders an unordered list', () => {
+    const list = mountWithAppProvider(<List {...mockProps} />);
+    expect(list.find('ul')).toHaveLength(1);
+  });
+
+  describe('focusIndex', () => {
+    it('passes focusIndex to correct Item', () => {
```

Comment:
I would divide this into two tests `doesn't focus items by default` and `focusses the item at focusIndex`.